### ISSUE TITLE
Option to receive all call results as `Result` objects

### ISSUE THREAD

**Issue by oberstet** [crossbario/autobahn-js#55]
Currently, a call result is wrapped into `Result` if and only if the result has more than one positional result and/or a keyword result.

Consequently, user code expecting to receive complex results (or mixed simple and complex results) will need to test for type of result.

We might add a global option that will make _all_ results (simple or not) wrapped in `Result`.

**Comment by sundbry** [comment]
Yes, please implement this! It is troublesome when working with calls that return variable length lists. Right now we are forced to wrap them in a keyword result.
